test.rb

```diff
@@ -0,0 +1,36 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+def a
+  def b
+    1
+    ['a','c']
+    'C'
+    "A"
+    ?a
+  end
+end
+
+aaa;
+
+not 1
```

Comment:
__[development][rubocop]__
* Use `!` instead of `not`. [:ref](https://github.com/bbatsov/ruby-style-guide#bang-not-not)

<!-- Comment by [SideCI](https://sideci.com) -->